github-orgs/kubeflow/org.yaml

```diff
@@ -947,12 +947,14 @@ orgs:
           release-team:
             description: Kubeflow Release Team
             members:
-            - anencore94
             - annajung
+            - Davidnet
             - DomFleischmann
             - DnPlas
             - jbottum
+            - juliusvonkohout
             - kimwnasptd
+            - ryansteakley
             - surajkota
```

Comment:
@DnPlas Could you help add me as the release team member in this PR as well? Thx.